### DISCUSSION TITLE
I can't get two containers to see each other when routed through Gluetun

### DISCUSSION THREAD

**Discussion by ab3lson** [qdm12/gluetun#537] [Q&A] [answered]
I have two containers (Radarr and Jackett) running through Gluetun with the network_mode: service:gluetun, but I can't get Radarr and Jackett to talk to each other. Is there some sort of variable that I can set in my docker-compose for Gluetun to allow containers to talk to each other?

**Comment by qdm12** [comment]
If Radarr and Jackett are using gluetun as their network stack, they are on the same network host (gluetun) so they have to reach each other through 127.0.0.1:somePort

**Reply by ab3lson** [reply]
I figured it out! I was using the ports that I was exposing to the host and not the ports exposed to Gluetun.

**Comment by qdm12** [comment, marked as answer]
Just for future readers, selected answer from @ab3lson:

> I was using the port numbers published to the host and not the internal ports exposed to Gluetun.

**Comment by sebtech33** [comment]


**Reply by qdm12** [reply]
Conversation moved to #555 please check it, I'll remove the messages here.

**Comment by p6002** [comment]
Im using correct ports, and still cannot link lidarr to qbittorrent in gluetun stack
